sys/include/atomic_utils.h

```diff
@@ -753,9 +774,31 @@ static inline atomic_bit_u32_t atomic_bit_u32(volatile uint32_t *dest,
  * @brief   Create a reference to a bit in an `uint64_t`
  * @param[in]       dest        Memory containing the bit
  * @param[in]       bit         Bit number (`0` refers to the least significant)
+ *
+ * @return Opaque reference to the bit.
  */
 static inline atomic_bit_u64_t atomic_bit_u64(volatile uint64_t *dest,
                                               uint8_t bit);
+
+/**
+ * @brief   Create a reference to a bit in an `unsigned int`
+ * @param[in]       dest        Memory containing the bit
+ * @param[in]       bit         Bit number (`0` refers to the least significant)
+ *
+ * @return Opaque reference to the bit.
+ */
+static inline atomic_bit_unsigned_t atomic_bit_unsigned(volatile unsigned *dest,
+                                                        uint8_t bit)
+{
+#if UINT_MAX == UINT16_MAX
+    return atomic_bit_u16(dest, bit);
```

Comment:
```suggestion
    return atomic_bit_u16((volatile uint16_t *)dest, bit);
```

On 16 bit platforms the same issue is with `uint16_t`, as there both `unsigned int` and `unsigned short` are 16 bit and `uint16_t` can be either.